_test_null_safety/test/opted_in_test.dart

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:test/test.dart';
+
+import 'common/message.dart';
+
+void main() {
+  test('can use null safety', () {
+    expect(message!.isNotEmpty, true);
```

Comment:
[nit] I know that the intention of the test is to use a `!` but I always get nervous to see non-idiomatic code spread. If this were a real test we'd write it as `expect(message, isNot(isEmpty)`... I wonder if there is a more idiomatic test case we could write that would still include a `!`?

Reply:
🤷‍♂️ I don't think it really matters. I could change this to an assert.